docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -251,6 +251,9 @@ We need to map that to coala's convention as follows:
             normalize_column_numbers = True,
             ...)
 
+This is reminder for developers that they need to do research on linter bear,
```

Comment:
As @frextrite suggested, it should be `a reminder`.
Plus, how about `... to do a proper research on ...`?